src/webapp/components/dataset-wizard/FilterIndicators.tsx

```diff
@@ -28,6 +28,7 @@ export type FilterIndicatorsProps = {
     theme: string;
     types: string[];
     selectedType: string;
+    hide: boolean;
```

Comment:
Not sure of the semantics here, so just as a note, check if "hidden" is more idiomatic.